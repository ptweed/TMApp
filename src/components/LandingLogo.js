import React from 'react'
import { StyleSheet, View, Text } from 'react-native'
import { FontAwesome5 } from '@expo/vector-icons'
import Captions from '../utils/Captions'

const LandingLogo = () => (
    <View style={styles.iconContainer}>
        <Text style={styles.headerTitle}>{Captions.appName}</Text>
        <FontAwesome5 name="tasks" size={60} />
    </View>
)

const styles = StyleSheet.create({
    iconContainer: {
      backgroundColor: '#fff',
      alignItems: 'center',
      justifyContent: 'center',
      marginTop: 10,
      marginBottom: 10,
    },
    headerTitle: {
        margin:10,
        fontSize: 40,
    }
  })

export default LandingLogo