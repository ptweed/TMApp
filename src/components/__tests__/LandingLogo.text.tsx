import React from 'react'
import { render } from 'react-native-testing-library'
import LandingLogo from '../LandingLogo'

jest.mock('@expo/vector-icons', () => ({
    FontAwesome5: ''
}))
  
describe('LandingLogo Component', () => {
  it('LandingLogo renders correctly', () => {

    const formI = render(<LandingLogo/>).toJSON();
    expect(formI).toMatchSnapshot();

  });

});