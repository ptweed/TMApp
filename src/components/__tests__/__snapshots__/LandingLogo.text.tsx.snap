// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingLogo Component LandingLogo renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "justifyContent": "center",
      "marginBottom": 10,
      "marginTop": 10,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 40,
        "margin": 10,
      }
    }
  >
    Task Master
  </Text>
  <
    name="tasks"
    size={60}
  />
</View>
`;
